kinto_fxa/authentication.py

```diff
@@ -122,6 +141,11 @@ def _get_cache(self, request):
 
         return self._cache
 
+    def callback(self, userid, request):
+        if request.bound_data[KEY][1] != "default":
```

Comment:
no safety check here?

Reply:
Is this adding additional principals, beyond the userid?

Reply:
Yes